app/models/manageiq/providers/nuage/network_manager/event_catcher/stream.rb

```diff
@@ -2,17 +2,18 @@ class ManageIQ::Providers::Nuage::NetworkManager::EventCatcher::Stream
   include Vmdb::Logging
 
   def self.test_amqp_connection(options = {})
+    return false if options[:urls].empty?
```

Comment:
If making such test, you should also check that `:urls` actually exists in `options`?

Reply:
Done, thanks. Can't hurt to be onehundertpercent certain :)